deployment/clean_all.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Execute every clean.sh scripts in every folder.
+
+for directory in */ ; do
+    echo "Processing ${directory}"
+    pushd "$directory"
+    if [ -f clean.sh ]; then
+        ./clean.sh 
```

Comment:
I think it should clean up the `pkg-dist` tars too, to avoid leaving cruft around. OR, the `collect_all` script should `mv` instead of `cp`.

Reply:
Agreed. Will add (in `build.common.sh`)